damus.xcodeproj/project.pbxproj

```diff
@@ -1027,6 +1031,7 @@
 				ONLY_ACTIVE_ARCH = YES;
 				SDKROOT = iphoneos;
 				SWIFT_ACTIVE_COMPILATION_CONDITIONS = DEBUG;
+				SWIFT_EMIT_LOC_STRINGS = YES;
```

Comment:
Xcode 13 seems to turn this on by default for new projects, but Damus didn't have it, so I turned it on.
https://developer.apple.com/documentation/xcode-release-notes/xcode-13-release-notes

> The new Use Compiler to Extract Swift Strings build setting invokes the Swift compiler to accurately extract string interpolations and string literals from Text(), String(localized:), AttributedString(localized:) initializers, SwiftUI’s LocalizedStringKey, and Foundation’s StringLocalizationKey. Enabling this setting compiles all localized targets in your project when exporting and importing for localization. This build setting is enabled by default on all new projects, but can be opted-in for existing Swift projects. (58023398)